pkg/clients/instance.go

```diff
@@ -108,10 +108,17 @@ func NeedsSecretRotation(instance Instance) bool {
 	return instance.GetStatus().GetSynchronizationSecretName() != GetSecretName(instance)
 }
 
-func SetIDportenClientDefaultValues(in *naisiov1.IDPortenClient, cfg *config.Config) {
-	var defaultValidIdportenScopes = []string{"openid", "profile"}
-	var defaultValidKrrScopes = []string{"krr:global/kontaktinformasjon.read", "krr:global/digitalpost.read"}
+func GetIDPortenDefaultScopes(integrationType string) []string {
+	switch integrationType {
+	case string(types.IntegrationTypeIDPorten):
+		return []string{"openid", "profile"}
+	case string(types.IntegrationTypeKrr):
+		return []string{"krr:global/kontaktinformasjon.read", "krr:global/digitalpost.read"}
+	}
+	return []string(nil)
```

Comment:
Returning `[]string(nil)` has some consequences for clients that specify other integration types (such as `api_klient`) without any declared scopes. 

A nil slice is not the same as an empty slice. JSON marshalling for instance will give different results:

```golang
data := []string(nil) 
// "data": null

data := []string{} 
// "data": []
```

I think `api_klient` clients should have the same set of default scopes as an `idporten` client as well.

In summary:

```suggestion
	switch integrationType {
	case string(types.IntegrationTypeIDPorten), string(types.IntegrationTypeApiKlient):
		return []string{"openid", "profile"}
	case string(types.IntegrationTypeKrr):
		return []string{"krr:global/kontaktinformasjon.read", "krr:global/digitalpost.read"}
	}
	return []string{}
```

I've pushed some tests for these changes at the `master` branch.

Reply:
fixed 👍